SU-HSDO/stanford_mrc
# MRCD8-201 Event Fixes

# READY FOR REVIEW

# Summary
- Fixed the issue with emdash separator and the admission information being left blank
- Updated the Flexbox display for the pipe between time and location now that the "|" is added to the template directly

# Needed By (Date)
- End of sprint (1/17)

# Urgency
- N/A

# Steps to Test

1. Pull this branch
2. Test that an event without admission information present still has a trailing emdash on the line below it (rather than inline with it)
3. Verify that the "|" character between time and location on the event card wraps nicely (pipe should not sit on its own line, but rather always be to the right of the time)

# Affected Projects or Products
- stanford_mrc

# Associated Issues and/or People
- https://stanfordits.atlassian.net/browse/MRCD8-201

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
